docker-compose.yml

```
@@ -0,0 +1,12 @@
+version: "3"
+services:
+  actual_server:
+    container_name: actual_server
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - "5006:5006"
+    volumes:
+      - ./data:/data
```

```suggestion
      - ./server-files:/app/server-files
      - ./user-files:/app/user-files
```

I think this is what you want for the volumes. 

I'm matches these in the `Dockerfile`. 

```Dockerfile
RUN mkdir ./server-files
RUN mkdir ./user-files
```